### DISCUSSION TITLE
About adaptability and responsiveness

### DISCUSSION THREAD

**Discussion by Rainproof** [frangoteam/FUXA#384] [General]
I think the theme of responsiveness and adaptability is relevant.
Suppose someone needs a SCADA system with details and large mnemonic diagrams, and someone is developing a web application. which will be used from phones and tablets.
I'm not talking about completely rebuilding the concept of FUXA only for responsiveness and adaptability. But only to have separate types of views with properties of responsiveness, and the property of autoscalling the widget.

**Comment by unocelli** [comment]
we can try to allow the view type to 'multi view container' with the fit to screen [option](https://tiberiuzuld.github.io/angular-gridster2/)

**Comment by Rainproof** [comment]
Yes, I understand that it is almost impossible to make the behavior of elements that the user randomly sets to view responsive. When the developer does this, he also describes the styles after setting the element to a certain position in the DOM .

**Reply by unocelli** [reply]
the problem is that in fuxa views are svg images, so html components such as buttons are inserted into the svg with the svg:foreignObject and not as a normal dom of an html page.

**Comment by Rainproof** [comment]
Hi,
In principle, if the user had the opportunity to combine the mnemonic diagram into a container in the view, and assign the "auto-scaling" behavior to it. For other elements, allow the user to assign some other behavior to them. Well, something like this )  . These are my fantasies, sorry)
